addons/medical_status/functions/fnc_isInStableCondition.sqf

```diff
@@ -17,7 +17,31 @@
 
 params ["_unit"];
 
-alive _unit
-&& {!IS_UNCONSCIOUS(_unit)}
-&& {GET_WOUND_BLEEDING(_unit) == 0}
-&& {_unit call FUNC(hasStableVitals)}
+if (!alive _unit
+    || {IS_UNCONSCIOUS(_unit)}
+    || {GET_HEART_RATE(_unit) < EGVAR(medical_treatment,minHeartRatePAK)}
+) exitWith {false};
+
+private _requiredBloodVolume = DEFAULT_BLOOD_VOLUME * (EGVAR(medical_treatment,minBloodPAK) / 100);
+if (GET_BLOOD_VOLUME(_unit) < _requiredBloodVolume) exitWith {false};
```

Comment:
This'd be more intuitive if it used qualitative blood loss (hemmorhage class, "Lost some/a lot of/etc blood") instead.